### ISSUE TITLE
How to install

### ISSUE THREAD

**Issue by salehi** [kochsebastian/ChessVisionBot#4]
Could you provide a HowToInstall document?
I can't install conda requirements

```conda
(/app) root@08bc4e638069:/app# conda install --file requirements.txt -c conda-forge -c defaults
WARNING: The conda.compat module is deprecated and will be removed in a future release.
Collecting package metadata: done
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - python-chess==0.30.0=pypi_0
  - libtiff==4.1.0=hcb84e12_0
  - ruamel_yaml==0.15.87=py37h1de35cc_0
  - tk==8.6.8=ha441bb4_0
  - libiconv==1.15=hdd342a3_7
  - libpng==1.6.37=ha441bb4_0
  - conda-package-handling==1.6.0=py37h0b31af3_1
  - grpcio==1.16.1=py37h044775b_1
  - ncurses==6.1=h0a44026_1
  - pcre==8.43=h0a44026_0
  - xz==5.2.4=h1de35cc_4
  - wrapt==1.11.2=py37h1de35cc_0
  - libprotobuf==3.11.2=hd9629dc_0
  - numpy-base==1.17.4=py37ha711998_0
  - tensorflow-base==1.15.0=mkl_py37h032239d_0
  - pyyaml==5.2=py37h1de35cc_0
  - expat==2.2.6=h0a44026_0
  - cairo==1.14.12=hc4e6be7_4
  - pyscreenshot==0.6=pypi_0
  - jpeg==9b=he5867d9_2
  - graphite2==1.3.13=h2098e52_0
  - pyobjc-framework-cocoa==5.2=py37_0
  - tornado==6.0.3=py37h1de35cc_0
  - pyobjc-framework-quartz==5.2=py37_0
  - zstd==1.3.7=h5bba6e5_0
  - libvpx==1.7.0=h378b8a2_0
  - fribidi==1.0.5=h1de35cc_0
  - mss==3.0.1=pypi_0
  - scipy==1.3.1=py37h1a1e112_0
  - matplotlib==3.1.1=py37h54f8f79_0
  - numpy==1.17.4=py37hfd23ef0_0
  - pycosat==0.6.3=py37h0b31af3_1002
  - py-opencv==3.4.2=py37h7c891bd_1
  - libopencv==3.4.2=h7c891bd_1
  - openssl==1.1.1d=h1de35cc_3
  - glib==2.63.1=hd977a24_0
  - pyobjc-core==5.2=py37_0
  - libffi==3.2.1=h475c297_4
  - freetype==2.9.1=hb4e5f40_0
  - cryptography==2.8=py37hafa8578_1
  - libgfortran==3.0.1=h93005f0_2
  - h5py==2.8.0=py37h878fce3_3
  - pyscreen==0.3=pypi_0
  - tensorflow==1.15.0=mkl_py37hb249377_0
  - harfbuzz==1.8.8=hb8d4a28_0
  - gettext==0.19.8.1=h15daf44_3
  - jasper==2.0.14=h636a363_1
  - libedit==3.1.20181209=hb402a30_0
  - readline==7.0=h1de35cc_5
  - libopus==1.3=h1de35cc_0
  - hdf5==1.10.2=hfa1e0ec_1
  - python==3.7.5=h359304d_0
  - pango==1.42.4=h060686c_0
  - bzip2==1.0.8=h1de35cc_0
  - libopenblas==0.3.6=hdc02c5d_2
  - yaml==0.1.7=hc338f04_2
  - sqlite==3.30.1=ha441bb4_0
  - cffi==1.13.2=py37h33e799b_0
  - fontconfig==2.13.0=h5d5b041_1
  - easyprocess==0.2.8=pypi_0
  - icu==58.2=h4b95b61_1
  - c-ares==1.15.0=h1de35cc_1001
  - protobuf==3.11.2=py37h0a44026_0
  - libcxx==4.0.1=hcfea43d_1
  - libxml2==2.9.9=hf6e021a_1
  - pillow==6.2.1=py37hb68e598_0
  - graphviz==2.40.1=hefbbd9a_2
  - libcxxabi==4.0.1=hcfea43d_1
  - zlib==1.2.11=h1de35cc_3
  - ffmpeg==4.0=h01ea3c9_0
  - kiwisolver==1.1.0=py37h0a44026_0
  - pixman==0.38.0=h1de35cc_0

Current channels:

  - https://conda.anaconda.org/conda-forge/linux-64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/free/linux-64
  - https://repo.anaconda.com/pkgs/free/noarch
  - https://repo.anaconda.com/pkgs/r/linux-64
  - https://repo.anaconda.com/pkgs/r/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.


(/app) root@08bc4e638069:/app# 
```

**Comment by kochsebastian** [comment]
No a how to install is not in my interest. This basically serves as a safeguard against people who don't code and just want to use this project to cheat.
It's an unusual problem you have, you could try to use the yaml environment file which contains more conda metadata.

**Comment by salehi** [comment]
Dude! its github, not cheathub!

Somebody like me, is trying to help your project to be more comfortable to everyone, Although you don't want to.

Good Luck

**Comment by kochsebastian** [comment]
I appreciate your feedback.
I gave you an alternate way of installing the dependencies, besides that I would expect that a Python programmer would be able to install the dependencies manually although I admit that wouldn't be ideal.
But I stand to my way of not publishing 'How To' instructions. I noticed that in some chess forums my chess bot is indeed mentioned as a good way to cheat and how to set it up and use it. I don't want  to give them an easier way to cheat. As a real programmer it's easy too set up this project even if it's maybe not a smooth operation but every programmer should be able to solve these issues as thy happen on other projects too (For real problems with for example TensorFlow model problems, see issue [Saved model file does not exist](https://github.com/kochsebastian/ChessVisionBot/issues/5) I gladly help). 
This project is designed to showcase computer vision and machine learning in a chess environment, NOT TO CHEAT.
